src/main/java/command/PrintAreaLocationsCommand.java

```diff
@@ -0,0 +1,54 @@
+package command;
+
+import eventlist.EventList;
+import locationlist.BusStopList;
+import locationlist.LocationList;
+import storage.Storage;
+import ui.UI;
+
+/**
+ * Represents the command for printing locations available at some areas in NUS.
+ */
+public class PrintAreaLocationsCommand extends Command {
+    String area;
+
+    public PrintAreaLocationsCommand(String area) {
+        this.area = area;
+    }
+
+    /**
+     * Purely print the available locations at the area selected by the user.
+     *
+     * @param events    the list of Events.
+     * @param locations the list of Locations.
+     * @param busStops  the list of BusStops.
+     * @param ui        do outputs.
+     * @param storage   store the data.
+     */
+    @Override
```

Comment:
I like how you style the code into a simple and straight-forward manner, as well as the JavaDoc, with brief description and everything labelled clearly.